wee_slack.py

```diff
@@ -1046,6 +1046,16 @@ def get_channel_map(self):
     def get_username_map(self):
         return {v.slack_name: k for k, v in self.users.iteritems()}
 
+    def find_channel_by_name(self, name):
+        for channel in self.channels.itervalues():
+            if channel.slack_name == name:
+                return channel
```

Comment:
This method can be written like this, instead of looping over the channels:
```
channel_key = self.get_channel_map().get(name)
return self.channels.get(channel_key)
```

Reply:
Technically `get_channel_map` loops over the channels also, it just wouldn't be done explicitly here I guess. I just considered this a tiny optimization since it won't always loop over all channels and won't create a dict that'll just get used once and then thrown away. But if readability is a bigger concern I can definitely change it!

Reply:
> Technically get_channel_map loops over the channels also, it just wouldn't be done explicitly here I guess.

Yeah, true, but I don't think the performance difference matters in this case. Especially since it's only called after user action.